### ISSUE TITLE
[kpi] coarse-user-agent-parser needs to learn about Windows 8 (reports os: 'Unknown')

### ISSUE THREAD

**Issue by jrgm** [mozilla/persona#3187]
```
$ node 
> coarse = require('./lib/coarse_user_agent_parser') 
{ parse: [Function] }
> str = 'Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; WOW64; Trident/6.0)'
'Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; WOW64; Trident/6.0)'
> coarse.parse(str)
{ os: 'Unknown', browser: 'MSIE', version: 10 }
```

See https://github.com/mozilla/browserid/blob/dev/lib/coarse_user_agent_parser.js#L25. There needs to be a matcher for 'Windows NT 6.2'.

**Comment by jaredhirsch** [comment]
idk the source of our UA parser but this one's based on Browserscope data and might be worth considering: https://github.com/3rd-Eden/useragent

**Comment by jrgm** [comment]
Yeah, perhaps a third-party library would ease maintenance, but for now, to get 'Windows 8', I fixed what's there.

**Comment by jrgm** [comment]
Pull request landed to recognize Windows 8 (Windows NT 6.2). Closing.

**Comment by jaredhirsch** [comment]
actually closing
